data/tests/html/aria/landmarks-aria.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>ARIA landmarks test suite</title>
+  <style type="text/css">
+    div { border: 1px solid darkblue; padding: 10px }
+  </style>
+</head>
+
+<body>
+  <div role="banner">
+    <p>This is the start of the banner role container</p>
+    <p><a href="/">A11ySupport home</a></p>
+    <div role="search">
+      <p>This is the start of a search role container</p>
+      <label for="txtSearch">Search A11ySupport</label>
+      <input id="txtSearch" type="search" />
+      <p>This is the end of a search role container</p>
+    </div>
+    <p>This is the end of the banner role container</p>
+  </div>
+  <div role="navigation" aria-label="Site sections">
```

Comment:
Do you think it would be good to test named landmarks vs unnamed landmarks? I wonder if naming would affect behavior. Maybe we could create an example where all the landmarks are unnamed, then another where they are all named? That could be a lot of testing...

Reply:
Up to you - if you think there's benefit in adding another (unnamed) navigation container and testing, let me know. Would that be a separate "conveys role when unnamed" expectation or the existing generic "conveys role" expectation?

Reply:
I think it would be worth it to test both named and unnamed, especially because most pages will have an unnamed role. The expectation is the same in both cases: the role must be conveyed. At least, I think that's the expectation... testing should reveal if it is not, and then we might need to revisit the expectations.

Reply:
Although, I'd suggest making two HTML files. One where all landmarks are named, and another where all are unnamed. Rather than combine the two into one... most pages will likely only have one of each kind of landmark here.

Reply:
Added unnamed navigation role test suite, using another HTML file as suggested.